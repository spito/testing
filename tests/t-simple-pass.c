#include <cut.h>

TEST(success,
    TEST_SETTINGS(TEST_POINTS(1)))
{
    ASSERT(1);
}

TEST(suppressed, TEST_SETTINGS(TEST_SUPPRESS)) {
    ASSERT(0);
}