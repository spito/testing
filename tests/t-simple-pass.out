[  1] success................................................................OK
[  2] suppressed.....................................................SUPPRESSED

Summary:
  tests:          2
  succeeded:      1
  filtered out:   0
  suppressed:     1
  skipped:        0
  failed:         0

Points:           1.00
